plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudget.hs

```diff
@@ -178,10 +178,16 @@ instance ExBudgetBuiltin fun () where
 
 data ExBudget = ExBudget { _exBudgetCPU :: ExCPU, _exBudgetMemory :: ExMemory }
     deriving stock (Eq, Show, Generic, Lift)
-    deriving (Semigroup, Monoid) via (GenericSemigroupMonoid ExBudget)
```

Comment:
`GenericSemigroupMonoid` doesn't derive the right thing for `stimes`?

Reply:
It does... but it goes via `rep` etc. This version gets inlined into direct multiplications.